hamilflow/models/brownian_motion.py

```diff
@@ -86,8 +87,8 @@ class BrownianMotion:
     \end{align}
     $$
 
-    References:
-
+    References
+    ----------
```

Comment:
~~Is this supposed to be a title? This is the rst syntax. We are using markdown. This will generate a horizontal line.~~

Reply:
![Screenshot_20240828-073703.png](https://github.com/user-attachments/assets/06478b6a-4978-41c4-a939-9d4300e5d234)

Oh I was wrong. Interesting, didn't know this also works as a heading.